packages/complex-rubric/controller/src/index.js

```diff
@@ -88,3 +89,119 @@ export const createCorrectResponseSession = (question, env) => {
     }
   });
 };
+
+// this should always be updated when validate function of rubric is changed (as for ebsr)
+const validateSimpleRubric = (model) => {
```

Comment:
“IMPORTANT! This function is a duplicate of rubric/controller/validate function! If you make any changes to this function, please make sure you also update rubric/controller/validate function!“.